orca-loadtest/src/main/scala/com/netflix/spinnaker/orca/loadtest/OrcaSimulation.scala

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2017 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.orca.loadtest
+
+import com.netflix.spinnaker.orca.loadtest.scenarios.OrcaScenarios
+import io.gatling.core.Predef._
+import io.gatling.core.feeder.RecordSeqFeederBuilder
+import io.gatling.core.structure.PopulationBuilder
+import io.gatling.http.Predef._
+
+import scala.collection.mutable.ListBuffer
+import scala.concurrent.duration._
+
+class OrcaSimulation extends Simulation {
+
+  val config = new OrcaSimulationConfig(OrcaSimulationConfig.loadConfig())
+
+  // TODO rz - Make a circular feeder of RawFileBody instead.
+  lazy val CircularTaskFeeder: Feeder[String] = {
+    Iterator.continually(Map(
+      "body" -> ""
+    ))
+  }
+
+  def dataSetup(): Unit = {
```

Comment:
If you don't think you need it, I'd remove this - helps declutter things.  You can always add it back later.